Arriba/Arriba.Web/configuration.default/Configuration.jsx

```diff
@@ -48,7 +48,7 @@
             overview:(
                 <span>
                     Arriba is a data exploration, free text search, and structured search engine.
-                    Need more <a href="Search.html?help=true">help</a>?
+                    Need more <a href="/?help=true">help</a>?
```

Comment:
Unrelated: I think there's a bug currently where it's hard to tell how to "close" help again. Could you double check? We should potentially make clicking the help button again or clicking an 'X' in the corner take you back to the start screen. Low Priority, since people could just close the browser.